src/debug/debug.cpp

```diff
@@ -73,6 +78,23 @@ static void DrawVariables(void);
 char* AnalyzeInstruction(char* inst, bool saveSelector);
 uint32_t GetHexValue(char* str, char*& hex);
 
+string get_full_path(const string& rel_or_abs_path) 
```

Comment:
I think this whole implementation can be replaced by [`std::filesystem::absolute`](https://en.cppreference.com/w/cpp/filesystem/absolute)?

Aside from that, please mark any function intended to be static (like this one) as `static` - it helps a lot.